docs/example-lpa.json

```diff
@@ -1,42 +1,43 @@
 {
+  "lpaType": "personal-welfare",
   "donor": {
-    "firstNames": "Homer",
-    "surname": "Zoller",
-    "dateOfBirth": "1960-04-06",
-    "email": "h.zoller@example.com",
+    "firstNames": "Feeg",
+    "lastName": "Bundlaaaa",
     "address": {
-      "line1": "79 Bury Rd",
-      "town": "Hampton Lovett",
-      "postcode": "WR9 2PF",
+      "line1": "74 Cloob Close",
+      "town": "Mahhhhhhhhhh",
       "country": "GB"
-    }
```

Comment:
This highlighted a bug: postcode should be required if country == GB. I've created [a bug ticket for that](https://opgtransform.atlassian.net/browse/VEGA-2317).

Was there a reason to change this otherwise? Doesn't really matter, just interested.

Reply:
@gregtyler it was just that I created my own example JSON, then realised that there was already an example in the repo; I just overwrote what was there with my own